docs/install.md

```diff
@@ -182,25 +185,16 @@ data:
   bucket.service.account.field.name: BOTO_CONFIG
 ```
 
-Both options provide the same functionality to the pipeline. The choice is based
-on the infrastructure used, for example in some Kubernetes platforms, the
-creation of a persistent volume could be slower than uploading/downloading files
-to a bucket, or if the the cluster is running in multiple zones, the access to
-the persistent volume can fail.
-
-### Overriding default ServiceAccount, Timeout or PodTemplate used for TaskRun and PipelineRun
+## Customizing basic execution parameters
 
-The ConfigMap `config-defaults` can be used to override default values.
-You can override the default service account, the default timeout, and the
-default pod template applied to `TaskRun` and `PipelineRun`.
+You can specify your own values that replace the default service account (`ServiceAccount`), timeout (`Timeout`), and Pod template (`PodTemplate`) values used by Tekton Pipelines in `TaskRun` and `PipelineRun` definitions. To do so, use the ConfigMap `config-defaults`.
```

Comment:
Suggest "create the ConfigMap" at the end here rather than "use the ConfigMap".  Or maybe "modify"?  "Use" feels like the wrong verb to me hre since the operator will need to take a mutating action on the ConfigMap.

Reply:
Thank you! I don't know exactly what a ConfigMap is so I didn't know the correct verb to pair it with. Changed to modify and added a bit of context.